packages/react-components/react-components/src/index.ts

```diff
@@ -471,6 +471,7 @@ export type {
   PopoverSurfaceState,
   PopoverTriggerProps,
   PopoverTriggerState,
+  PopoverTriggerChildProps,
```

Comment:
can you provide more context why it should be exported ?

Reply:
Updated the PR description